app/models/application_record.rb

```diff
@@ -10,7 +10,7 @@ class ApplicationRecord < ActiveRecord::Base
   private
 
   def event_data
-    { table_name: self.class.table_name }.merge(anonymised_attributes)
+    { "table_name" => self.class.table_name }.merge(anonymised_attributes)
```

Comment:
Why this key should change to a symbol?

Reply:
It's merged with attributes, which are all string keys. It's less confusing when all the keys are the same type.